### DISCUSSION TITLE
Fetchmail wrong working

### DISCUSSION THREAD

**Discussion by PunkFleet** [docker-mailserver/docker-mailserver#4289] [Q&A]
I following this https://docker-mailserver.github.io/docker-mailserver/latest/config/advanced/mail-fetchmail/

but my local client 'thunderbird' get multi same email notification and not display date:
<img width="636" alt="image" src="https://github.com/user-attachments/assets/149f1c64-63c2-4852-ad26-21fccf416c88" />
and there is my config:
```
[root@mx1 ~]# cat  docker-data/dms/config/fetchmail.cf
set postmaster "postmaster@example.com"
set no bouncemail
poll 'mx1.example.com' proto imap
  user 'info@example2.com' pass 'passwd' is 'info@example2.com'
  user 'test@example2.com' pass 'passwd' is 'test@example2.com'
[root@mx1 ~]# 

```
mailserver.env
```
# -----------------------------------------------
# --- Fetchmail Section -------------------------
# -----------------------------------------------

ENABLE_FETCHMAIL=1

# The interval to fetch mail in seconds
FETCHMAIL_POLL=60
# Use multiple fetchmail instances (1 per poll entry in fetchmail.cf)
# Supports multiple IMAP IDLE connections when a server is used across multiple poll entries
# https://otremba.net/wiki/Fetchmail_(Debian)#Immediate_Download_via_IMAP_IDLE
FETCHMAIL_PARALLEL=0

# Enable or disable `getmail`.
#
# - **0** => Disabled
# - 1 => Enabled
ENABLE_GETMAIL=0

# The number of minutes for the interval. Min: 1; Default: 5.
GETMAIL_POLL=5
```

**Comment by casperklein** [comment]
Did you try the [debug command](https://docker-mailserver.github.io/docker-mailserver/latest/config/advanced/mail-fetchmail/#debugging)?

Alternatively you could try getmail6 instead.

**Reply by PunkFleet** [reply]
> Did you try the [debug command](https://docker-mailserver.github.io/docker-mailserver/latest/config/advanced/mail-fetchmail/#debugging)?
> 
> Alternatively you could try getmail6 instead.

Im trying ,and also explain the log by chatGPT, but it's normal.
I After it was explained that since I'm only receiving emails in real time from the client, I'm perfectly fine with not using this service, so I gave up.
